docs/README.md

```diff
@@ -1,3 +1,157 @@
+    Long Ranger: modules/longranger.md
+    MALT: modules/malt.md
+    mOTUs: modules/motus.md
+    Salmon: modules/salmon.md
+    STAR: modules/star.md
+    TopHat: modules/tophat.md
+  Post-alignment:
+    Bamtools: modules/bamtools.md
+    Bcftools: modules/bcftools.md
+    biobambam2: modules/biobambam2.md
+    BUSCO: modules/busco.md
+    Bustools: modules/bustools.md
+    Conpair: modules/conpair.md
+    DamageProfiler: modules/damageprofiler.md
+    DeDup: modules/dedup.md
+    deepTools: modules/deeptools.md
+    Disambiguate: modules/disambiguate.md
+    EigenStratDatabseTools: modules/eigenstratdatabasetools.md
+    featureCounts: modules/featureCounts.md
+    fgbio: modules/fgbio.md
+    GATK: modules/gatk.md
+    GffCompare: modules/gffcompare.md
+    goleft_indexcov: modules/goleft_indexcov.md
+    gopeaks: modules/gopeaks.md
+    Hap.py: modules/happy.md
+    HiCExplorer: modules/hicexplorer.md
+    HOMER: modules/homer.md
+    HOPS: modules/hops.md
+    HTSeq: modules/htseq.md
+    JCVI: modules/jcvi.md
+    Kaiju: modules/kaiju.md
+    Kraken: modules/kraken.md
+    MACS2: modules/macs2.md
+    Methyl QA: modules/methylQA.md
+    mirtop: modules/mirtop.md
+    miRTrace: modules/mirtrace.md
+    mosdepth: modules/mosdepth.md
+    MTNucRatio: modules/mtnucratio.md
+    MultiVCFAnalyzer: modules/multivcfanalyzer.md
+    NanoStat: modules/nanostat.md
+    Nextclade: modules/nextclade.md
+    ngsderive: modules/ngsderive.md
+    odgi: modules/odgi.md
+    OptiType: modules/optitype.md
+    Pangolin: modules/pangolin.md
+    Peddy: modules/peddy.md
+    phantompeakqualtools: modules/phantompeakqualtools.md
+    Picard: modules/picard.md
+    Preseq: modules/preseq.md
+    Prokka: modules/prokka.md
+    PURPLE: modules/purple.md
+    qc3C: modules/qc3C.md
+    QoRTs: modules/qorts.md
+    Qualimap: modules/qualimap.md
+    Quast: modules/quast.md
+    RNA-SeQC: modules/rna_seqc.md
+    Rockhopper: modules/rockhopper.md
+    RSEM: modules/rsem.md
+    RSeQC: modules/rseqc.md
+    Sambamba: modules/sambamba.md
+    Samblaster: modules/samblaster.md
+    Samtools: modules/samtools.md
+    Sargasso: modules/sargasso.md
+    Sentieon: modules/sentieon.md
+    SexDetErrmine: modules/sexdeterrmine.md
+    Slamdunk: modules/slamdunk.md
+    Snippy: modules/snippy.md
+    SnpEff: modules/snpeff.md
+    SNPsplit: modules/snpsplit.md
+    Somalier: modules/somalier.md
+    Stacks: modules/stacks.md
+    Supernova: modules/supernova.md
+    THeTA2: modules/theta2.md
+    UMI-tools: modules/umitools.md
+    VarScan2: modules/varscan2.md
+    VCFTools: modules/vcftools.md
+    VEP: modules/vep.md
+    verifyBAMID: modules/verifybamid.md
+    WhatsHap: modules/whatshap.md
+Custom Content:
+  Introduction: custom_content.md
+Coding with MultiQC:
+  Writing new modules: modules.md
+  Plotting Functions: plots.md
+  MultiQC Plugins: plugins.md
+  Writing new templates: templates.md
+  Updating for compatibility: compatibility.md
+---
+
```

Comment:
This block was recently removed in the master branch, I think it's left here as a merge artefact :)

```suggestion
```